src/components/slider/Slider.test.tsx

```diff
@@ -33,3 +37,31 @@ describe('Slider component', () => {
     expect(await axe(container)).toHaveNoViolations()
   })
 })
+
+describe('Slider.Value component', () => {
+  it('renders', async () => {
+    const { container } = render(<Slider.Value value={[50]} />)
+
+    expect(container).toMatchSnapshot()
+  })
+
+  it('renders with a custom value label', async () => {
+    render(<Slider.Value value={[50]} outputLabel={outputLabel} />)
+
+    expect(await screen.findByText('50 test items')).toBeVisible()
+  })
+
+  it('renders with a custom empty value label', async () => {
+    render(<Slider.Value value={[0]} emptyData={emptyData} />)
+
+    expect(await screen.findByText('No test items')).toBeVisible()
```

Comment:
I think this very tiny change would make the relationship between the input and expected output a little more explicit:
```suggestion
    expect(await screen.findByText(emptyData.label)).toBeVisible()
```

Reply:
oh yeah, that makes a lot more sense! done.